.github/ISSUE_TEMPLATE/calamari_xcm_onboarding.md

```diff
@@ -108,6 +114,10 @@ console.log(tx.toHex());
         5. `DepositAsset: { assets: Wild { Wild: All }, maxAssets: 1, beneficiary: { parents: 0, interior: X1 { X1: AccountId32 { network: Any, id: SovereignAccountonRelay } } } }`
     
     **Note:** The values used above are for reference to be used in this testing environment, do not use these values in production!
+
+	**Note:** to watch out for having enough funds for the `WithdrawAsset` instruction and to reserve in the encoded HRMP calls.
```

Comment:
```suggestion
	**Note:** Ensure to have enough funds for the `WithdrawAsset` instruction and to reserve in the encoded HRMP calls.
```
not sure why github is rendering this more indented than the L116 note, it looks a bit weird in the review md

Reply:
Thanks.
Because it's tabs vs spaces. I think spaces are better, at least the last formatting problem was because of tabs again so I stopped using them here.